go.mod

```diff
@@ -17,11 +17,11 @@ require (
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240516150131-e1be553a9d10
 	github.com/smartcontractkit/libocr v0.0.0-20240326191951-2bbe9382d052
 	github.com/stretchr/testify v1.9.0
+	go.uber.org/multierr v1.11.0
```

Comment:
The linter should be flagging this and telling you to use `errors.Join` from std lib instead.

Is the linter broken again?!

Reply:
It is running but giving a false positive :facepalm:

Reply:
i was trying to avoid using it but it was causing issues in the core CI where a config test started to fail:

```
--- FAIL: TestConfig_Validate (0.00s)
    --- FAIL: TestConfig_Validate/invalid (0.00s)
        config_test.go:1327: 
            	Error Trace:	/home/runner/work/chainlink/chainlink/core/services/chainlink/config_test.go:1540
            	            				/home/runner/work/chainlink/chainlink/core/services/chainlink/config_test.go:1327
            	            				
  <truncated>
  
  
            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -73,5 +73,5 @@
            	            	 			- Nodes: missing: must have at least one node
            	            	-	- Solana: 5 errors:
            	            	+	- Solana: 4 errors:
            	            	 		- 1.ChainID: invalid value (mainnet): duplicate - must be unique
            	            	-		- 1.Nodes.1.Name: invalid value (bar): duplicate - must be unique
            	            	+	1.Nodes.1.Name: invalid value (bar): duplicate - must be unique
            	            	 		- 0.Nodes: missing: must have at least one node
   ```
   
   basically the error delimiter changed that I mentioned here: https://github.com/smartcontractkit/chainlink-solana/pull/709#discussion_r1605589134

Reply:
Hmmm but this output is not based on the formatted errors, it is based on trying to apply a custom format. I think we are probably hard coded for the old multierr types instead of using the interface implemented by the std lib. Can we fix that in common/core?

Reply:
core just uses multierr.Errors to unwrap everything: https://github.com/smartcontractkit/chainlink/blob/0d58a8d5db24f42720226e73328e501637ba59c5/core/utils/errors.go#L17

I'm mostly confused why this is happening since errors.Join should provide `Unwrap() []error` which should break down things properly within multierr.Errors